kommandir/inventory/group_vars/openstack/creation_destruction.yml

```diff
@@ -41,11 +41,12 @@ cloud_provisioning_command:
         bin/openstack_discover_create.py \
             {{ "--verbose" if adept_debug == True else "" }} \
             {{ "--lockdir=" ~ hostvars.kommandir.global_lockdir | trim if hostvars.kommandir.global_lockdir|default() not in empty else ""}} \
-            --image {{ peon_image | trim }} \
-            --flavor {{ peon_flavor | trim }} \
+            --image={{ peon_image | trim }} \
+            --flavor={{ peon_flavor | trim }} \
             {{ "--private" if public_peons is undefined or public_peons != True else "" }} \
-            {{ "--size " ~ peon_size | trim if peon_size | default("",True) not in empty else "" }} \
-            {{ "--userdata " ~ peon_userdata | trim if peon_userdata is defined else "" }} \
+            {{ "--size=" ~ peon_size | int if peon_size | default("",True) not in empty else "" }} \
+            {{ "--userdata=" ~ peon_userdata | int if peon_userdata is defined else "" }} \
```

Comment:
Why the change to `int`? In particular, I can't figure out what `peon_userdata` is supposed to be but as best I can tell it looks like a string? Unless I'm missing something major, command-line args are always strings, and it seems best not to try to second-guess what the type is supposed to be.

Reply:
Nope, it was a typo, thx for pointing it out fixed.